react/components/Statement/index.tsx

```diff
@@ -1,38 +1,49 @@
 import React from 'react'
 
 import SubjectAtom, { SubjectOptions } from './Atoms/SubjectAtom'
-import VerbAtom from './Atoms/VerbAtom'
+import VerbAtom, { VerbOption } from './Atoms/VerbAtom'
 import ObjectAtom from './Atoms/ObjectAtom'
+import { StatementProp } from './typings'
 
-type Props = {
+type StatementProps = {
   isFullWidth?: boolean
   isRtl?: boolean
   omitSubject?: boolean
   omitVerbs?: boolean
-  onChangeStatement: (statement: Props['statement']) => void
+  onChangeStatement: (statement: StatementProp) => void
   options: SubjectOptions
-  statement?: {
-    subject: string
-    verb: string
-    object?: unknown
-    error?: string
-  }
+  statement?: StatementProp
   subjectPlaceholder: string
 }
 
-const Statement: React.FC<Props> = ({
+const NoopComponent = () => <></>
+
+const Statement: React.FC<StatementProps> = ({
   isFullWidth = false,
   isRtl,
   omitSubject,
   omitVerbs,
   onChangeStatement,
   options,
-  statement = { subject: '', verb: '', object: null, error: null },
+  statement = { subject: '', verb: '', object: null },
   subjectPlaceholder,
 }) => {
-  const verbOptions =
-    statement.subject &&
-    options[statement.subject].verbs.find(verb => verb.value === statement.verb)
+  let verbOptions: VerbOption = {
+    label: '',
+    value: '',
+    object: NoopComponent,
```

Comment:
couldn't it be a 'null' here?

Reply:
Nope, because object is type `ObjectOption = (renderProps: RenderProps) => React.ReactElement`. Only if i change this `ObjectOption` type, but i think it's not a good idea.